cli/src/test/java/com/liferay/blade/cli/TestUtil.java

```diff
@@ -63,27 +63,34 @@ public static BladeTestResults runBlade(
 
 		StringBuilder sb = new StringBuilder();
 
+		boolean bridj = false;
+
 		try (Scanner scanner = new Scanner(error)) {
-			while (scanner.hasNextLine()) {
+			while (scanner.hasNextLine() && !bridj) {
 				String line = scanner.nextLine();
 
 				if ((line != null) && (line.length() > 0)) {
-					if (line.startsWith("SLF4J:")) {
-						continue;
+					if (line.contains("org/bridj/Platform$DeleteFiles")) {
```

Comment:
Does this code need to be in both TestUtil and in BladeTest?  Can't we just have this in one place?